src/annotator/guest.js

```diff
@@ -2,6 +2,9 @@ import scrollIntoView from 'scroll-into-view';
 
 import Delegator from './delegator';
 import { Adder } from './adder';
+import CrossFrame from './plugin/cross-frame';
+import DocumentMeta from './plugin/document';
+import PDFIntegration from './plugin/pdf';
```

Comment:
I plan to move and rename these files in future so that they no longer live in a "plugin" directory. That's out of scope for this PR though.